examples/python/README.md

```diff
@@ -20,19 +18,33 @@ pip install numpy
 pip install transformers
 pip install torch
 pip install onnx
-pip install onnxruntime
+pip install onnxruntime-gpu
 ```
 
+Choose a model. Examples of supported ones are:
+- Phi-2
+- Mistral
+- Gemma 2B IT
+- LLama 7B
+
 Run the model builder script to export, optimize, and quantize the model. More details can be found [here](../../src/python/py/models/README.md)
 
 ```bash
-cd examples/phi2/python
+cd examples/python
 python -m onnxruntime_genai.models.builder -m microsoft/phi-2 -e cpu -p int4 -o ./example-models/phi2-int4-cpu
 ```
 
-## Run the phi-2 model
+## Run the example model script
+
+To run the chat example...
+
+```bash
+python model-chat.py -m {path to model folder} -ep {cpu or cuda}
+```
+
+To run the token generation loop example...
 
 ```bash
-python phi2.py
+python model-generate.py -m {path to model folder} -ep {cpu or cuda} -i {string prompt}
 ```
 
```

Comment:
could you please add 2 explicit examples? one with phi2 for chat and one with phi2 for code generation.

Reply:
both examples are there, the other is underneath! i can join them into the same bash entry though

Reply:
how about add 2 concrete examples, like
python -m onnxruntime_genai.model.builder -m microsoft/phi2 -e cpu -p int4 models/phi2-cpu-int4
cd genai_models
python model-chat.py -m "models/phi2-cpu-int4" ...

Maybe create a .sh script to showcase the running of the model?